client/signup/steps/p2-get-started/style.scss

```diff
@@ -0,0 +1,61 @@
+.signup__step.is-p2-get-started {
+	color: var( --p2-color-white );
+	background-color: #000000;
+
+	.p2-step-wrapper__header-logo {
+		svg path {
+			fill: var( --p2-color-white );
+		}
+	}
+
+	.p2-get-started__option {
+		border-bottom: 1px solid #808080;
```

Comment:
Could we store these colors as variables, please?

Reply:
Done!